rotkehlchen/api/rest.py

```diff
@@ -2720,6 +2721,15 @@ def get_liquity_stability_pool_positions(self) -> dict[str, Any]:
     @async_api_call()
     def get_liquity_stats(self) -> dict[str, Any]:
         liquity_addresses = self.rotkehlchen.chains_aggregator.queried_addresses_for_module('liquity')  # noqa: E501
+        # make sure that all the entries that need it have the usd value queried
+        task_manager = self.rotkehlchen.task_manager
```

Comment:
Isn't this potentially really heavy? If there is a lot of events?

Reply:
is a price query that only affects the liquity events. In the best scenario nothing extra is queried, only the database. In the worst scenario you need the usd value of all the events (only liquity) but otherwise you get the wrong price